tests/test_project.py

```diff
@@ -1731,6 +1731,21 @@ def test_create_linked_view_heterogeneous_schema_problematic(self):
         with self.assertRaises(RuntimeError):
             self.project.create_linked_view(view_prefix)
 
+    def test_create_linked_view_with_slash_raises_error(self):
+        sp_badkey = [{'a/b': 0, 'b': 10} for i in range(5)]
+        sp_badkeyval = [{
+            'a/b': 'dir/file{}'.format(i), 'b': i % 3
```

Comment:
Yeah, that's a better idea. I've updated it so it will iterate through a list of bad characters and then the test also loops through that list, so adding new ones should be easy. I didn't account for nested keys or values, but I don't know if I should/I'm not sure the best way to do it.